packages/esm-patient-forms-app/src/forms/pagination-see-all.component.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Pagination from 'carbon-components-react/es/components/Pagination';
+import styles from '@openmrs/esm-patient-common-lib/src/pagination/pagination.component.scss';
+import { useTranslation } from 'react-i18next';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { usePaginationInfo } from '@openmrs/esm-patient-common-lib/src/pagination/usePaginationInfo';
```

Comment:
`@openmrs/esm-patient-common-lib` is fully imported. Just use `import { ...} from '@openmrs/esm-patient-common-lib'`.

Reply:
The corrections have been made.